src/py/aspen/database/models/__init__.py

```diff
@@ -1,4 +1,5 @@
 from .accessions import Accession, PublicRepository  # noqa: F401
+from .align_read import AlignRead, Bam  # noqa: F401
```

Comment:
did you also want to import `CallConsensus`? I didn't see it below

Reply:
Yup, I do.

Reply:
Fixed.